src/setup/plugin-multi-server-use/index.md

```diff
@@ -0,0 +1,32 @@
+# How to use MerginMaps plugin with multiple servers
```

Comment:
Mergin Maps Plugin for QGIS

How to Use Mergin Maps Plugin for QGIS with Multiple Servers